podpac/core/data/test/test_datasource.py

```diff
@@ -546,6 +546,8 @@ def test_eval_get_cache_extra_dims(self):
     def test_eval_get_cache_transform_crs(self):
         with podpac.settings:
             podpac.settings["DEFAULT_CACHE"] = ["ram"]
+            podpac.settings["DEFAULT_CRS"] = podpac.core.settings.DEFAULT_SETTINGS["DEFAULT_CRS"]
```

Comment:
@mpu-creare this was added as a precaution given that this function deals with CRS but it didn't resolve the test's failure.

This test still fails at ln560 `assert not node._from_cache` and while I understand that it shouldn't fail at that line (the node should not have originated from cache) I can't puzzle out the exact reason it fails.

@rparker-creare runs this test without it failing so I would suspect something related to python environment as the cause of that difference.

Reply:
Mostly like a difference in settings between you and @rparker-creare  -- I would start looking there.

Reply:
Probably related to one of the caching settings.

Reply:
The issue seems to be what DEFAULT_CRS setting you have in your `/home/<account>/.config/podpac/settings.json` file.

When I have `"EPSG:4326"`, the test fails but both `"+proj=longlat +datum=WGS84 +no_defs +vunits=m"` and `"EPSG:25832"` work.

@mpu-creare can you explain why might this be? My guess is it has something to do with some operations done using the default crs on initialization that are causing the nodes to be cached before expected (hence the test failure) but I don't understand enough about the crs' to guess what that might be.

Reply:
Hmm... this is a bit mysterious to me. I think it would require digging to figure out what's going on.

Reply:
Is resolving this something to be done before the release? Would it be improper for there be a means to prevent the default settings file adoption process and should the unit tests exercise it?

Reply:
This issue has been resolved. The problem was caused by another unit test creating the same node which caused it to be cached if the default CRS matched that of the test. The test has been changed to define a different node, preventing the issue regardless of CRS setting.